app/javascript/src/components/Dashboard/Contacts/utils.js

```diff
@@ -0,0 +1,25 @@
+import { CONTACTS_NAVLINKS } from "./constants";
+
+export const getActiveNavLink = key =>
+  CONTACTS_NAVLINKS.find(navlink => key === navlink.key);
+
+export const stringToDate = dateString => {
+  const monthNames = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "July",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec",
+  ];
+  const date = new Date(dateString);
+  return `${date.getDate()} ${
+    monthNames[date.getMonth()]
+  }, ${date.getFullYear()}`;
+};
```

Comment:
https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Prefer%20day.js%20over%20inbuilt%20date%20functions

Reply:
I have replaced the function. Thanks for pointing this out Yedhin.